src/openrct2/interface/Viewport.h

```diff
@@ -49,7 +49,7 @@ enum
     VIEWPORT_FLAG_TRANSPARENT_BACKGROUND = (1 << 19),
 };
 
-enum
+enum ViewportInteractionItem : uint8_t
```

Comment:
Do we want to help https://github.com/OpenRCT2/OpenRCT2/projects/14 and go for enum class now?

Reply:
I had a look at how much would be involved for this and decided to leave it for a follow on pr. Now that I've used the enum name it should be easier for anyone else to work out.

Reply:
Sure, sounds fine. I'll keep the comment unresolved so we can remember to open the issue later.